Updated lifecycle-management-overview.md (#126079)
Azure/azure-content: articles/storage/blobs/lifecycle-management-overview.md

You can enable last access time tracking to keep a record of when your blob is last read or written and as a filter to manage tiering and retention of your blob data. To learn how to enable last access time tracking, see [Optionally enable access time tracking](lifecycle-management-policy-configure.md#optionally-enable-access-time-tracking).

> When last access time tracking is enabled, the blob property called `LastAccessTime` is updated when a blob is read or written. A [Get Blob](/rest/api/storageservices/get-blob) &[Put Blob]_(/rest/api/storageservices/put-blob) operations is considered an access operation. [Get Blob Properties](/rest/api/storageservices/get-blob-properties), [Get Blob Metadata](/rest/api/storageservices/get-blob-metadata), and [Get Blob Tags](/rest/api/storageservices/get-blob-tags) aren't access operations, and therefore don't update the last access time. 

Reviewer: ```suggestion
When last access time tracking is enabled, the blob property called `LastAccessTime` is updated when a blob is read or written. [Get Blob](/rest/api/storageservices/get-blob) and [Put Blob]_(/rest/api/storageservices/put-blob) operations are considered access operations. [Get Blob Properties](/rest/api/storageservices/get-blob-properties), [Get Blob Metadata](/rest/api/storageservices/get-blob-metadata), and [Get Blob Tags](/rest/api/storageservices/get-blob-tags) aren't access operations, and therefore don't update the last access time. 
```

Reviewer: Thanks @Manjunath-microsoft - just a couple editorial edits. Please accept the edits

Author: Sure. Please proceed

Reviewer: Hi @Manjunath-microsoft - my apologies for not being clearer. You are the person who accepts the change. There should be some visible control somewhere around this edit that says "accept" or something similar.

Reviewer 2: @normesta @manjunath-microsoft I can commit the changes and process the PR.
